docs/install.rst

```diff
@@ -145,6 +127,21 @@ If you are used to starting Jupyter from `Terminal`, then remember to activate t
 That is all. Check out the :doc:`Tutorial </tutorial/index>` for some example code and Jupyter notebooks to get started.
 If importing the package fails please refer to the `Frequently asked questions`_ for troubleshooting options.
 
+Installation using pip
+----------------------
+
+.. _Python: https://www.python.org/
+.. _SciPy: http://www.scipy.org/
+
+Pylake can also be installed using pip with the following prerequisites:
+
+* `Python`_ 3.7 or newer (Python 2.x is not supported)
+* The `SciPy`_ stack of scientific packages
```

Comment:
having these lines here seems a bit weird since technically it's valid no matter how you install. Would probably add them back up at the top before the conda instructions

Reply:
When following the instructions under Anaconda, you don't need these prerequisites. I could also reformulate this sentence and leave it here?

Reply:
You can drop the SciPy prerequisite either way. No one needs to install it manually. I'd keep the Python version prerequisite since even with Anaconda you need to get a new-enough baseline installation.

Reply:
yeah maybe rewording is good then, because even with `pip` you don't need the scipy stack installed, as it's a requirement for `pylake`.

Reply:
I just noticed that under the current Anaconda instructions, Python 3.8 or newer is advised, while for pip 3.7 or newer is advised. Which of the two is correct?

Reply:
hmm interesting, maybe a typo. `pylake` supports `>=3.7` so we should stick with that